os/binfmt/libelf/libelf_bind.c

```diff
@@ -96,25 +96,55 @@ static int g_num_lib_syms;
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: elf_enable_caching
+ *
+ * Description:
+ *   Enable caching for large tables during relocation.
```

Comment:
As I know, using cache is only when we cannot allocate whole sym/reloc tables into the RAM.
If correct, could you leave the description that when we use the cache?
Then it is much better to understand.